Content.Shared/Paint/NoPaintShaderComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Paint;
+
+/// <summary>
+/// Applied to entites that should not have their layershaders altered such as Bar Signs and energy swords.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedPaintSystem))]
+public sealed partial class NoPaintShaderComponent : Component
+{
+}
```

Comment:
This should be a whitelist basis because this will just make yaml an unmaintainable nightmare.

Reply:
switched to using a blacklist to exlude (most things can be painted).